recipe/meta.yaml

```
@@ -1,34 +1,41 @@
 {% set name = "coin-or-utils" %}
-{% set version = "2.11.9" %}
+{% set version = "2.11.11" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/coin-or/CoinUtils/archive/releases/{{ version }}.tar.gz
-  sha256: 15d572ace4cd3b7c8ce117081b65a2bd5b5a4ebaba54fadc99c7a244160f88b8
+  sha256: 27da344479f38c82112d738501643dcb229e4ee96a5f87d4f406456bdc1b2cb4
   patches:
     - 0001-Patch-for-downstream.patch
 
 build:
   number: 0
-  skip: true  # [win]
   run_exports:
     - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
 
 requirements:
   build:
-    - gnuconfig  # [unix]
+    - gnuconfig
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('fortran') }}
-    - pkg-config
-    - make  # [unix]
+    - {{ compiler('fortran') }}  # [not win]
+    - pkg-config  # [not win]
+    - make  # [not win]
+    - m2-make  # [win]
+    - m2-pkg-config  # [win]
+    - m2-grep  # [win]
+    - m2-sed  # [win]
+    - m2-bash  # [win]
+    - m2-base  # [win]
+    
   host:
     - zlib
     - bzip2
-    - libblas
+    - blas  # for includes and .lib?
+    - libblas  # [not win]
```

What's the reason for adding a selector here? `libblas` is going to be present through `blas-devel` anyway; the only difference is whether the run-export is applied. But given that we seem to be building static libraries on windows here, there isn't even a link check complaint to motivate this?
```suggestion
    - libblas
```

---

Yes, so this was my last remaining doubt, too.
Basically I left the Linux and macOS builds "as they are", which means they will still build with fortran blas (I think).
On windows, due to the problems with fortran, I intended to use cblas.
I hope you know what I mean. I am happy to change it to "no Fortran" for all platforms and use cblas everywhere.

---

The problem is, though, that there apparently is no specific recipe for the headers and the .lib files of libcblas. So, I think I will need blas-devel and libcblas, right?
And then what? Will blas-devel pull in the Fortran libblas anyway?
Blas handling was quite confusing, maybe you could help out here, knowing what we want to do now?

---

All I can say is that I am pretty sure that just having libcblas did not work, at least on windows.

---

Yes, `libcblas` seems to only install a DLL and even that cannot be linked to (according to my understanding of linking on Windows, which is not great) because it is missing the `.dll.lib` that one usually needs at link time. Perhaps installing `blas-devel` fixes this. A static library would be better if that's possible. 

But note that we removed the `--with-blas` and so Blas is not currently being linked at all. If we get a `cblas` library that we think can be linked to successfully and the library is not in the default search path, we'll need to reintroduce the flags in order for `configure` to find it.

---

Resolved since I removed everything fortran related now.